compose.yml

```diff
@@ -23,9 +23,28 @@ services:
     volumes:
       - ./backend:/src/backend
 
+  database:
+    image: postgres
+    restart: always # Prevents stale state errors
+    environment:
+      POSTGRES_PASSWORD: cs222
+
+  pgadmin:
+    links:
+      - database:database
+    image: dpage/pgadmin4:3.0
+    environment:
+      PGADMIN_DEFAULT_EMAIL: 'admin@dishdiaries.com'
```

Comment:
you can probably use a secret file to apply the password and the emails for pgadmin so that we don't expose them here.